src/Api/Persistent.hs

```diff
@@ -261,7 +261,8 @@ nextId = do
     AUID <$> getDb dbLastId
 
 currentUser :: Persist (AUID User)
-currentUser = (\(Just u) -> u) <$> getDb dbCurrentUser
+-- currentUser = (\(Just u) -> u) <$> getDb dbCurrentUser
+currentUser = fromMaybe (AUID 0) <$> getDb dbCurrentUser
```

Comment:
This means that when logged out, user is admin by default.  I don't think we want this?